tools/src/main/java/org/apache/kafka/tools/ReplicaVerificationTool.java

```diff
@@ -0,0 +1,740 @@
+                .withOptionalArg()
+                .describedAs("hostname:port,...,hostname:port")
+                .ofType(String.class);
+            fetchSizeOpt = parser.accepts("fetch-size", "The fetch size of each request.")
+                .withRequiredArg()
+                .describedAs("bytes")
+                .ofType(Integer.class)
+                .defaultsTo(ConsumerConfig.DEFAULT_MAX_PARTITION_FETCH_BYTES);
+            maxWaitMsOpt = parser.accepts("max-wait-ms", "The max amount of time each fetch request waits.")
+                .withRequiredArg()
+                .describedAs("ms")
+                .ofType(Integer.class)
+                .defaultsTo(1_000);
+            topicWhiteListOpt = parser.accepts("topic-white-list", "DEPRECATED. Use --topics-include. " +
+                    "List of topics to verify replica consistency.")
+                .withRequiredArg()
+                .describedAs("Java regex (String)")
+                .ofType(String.class)
+                .defaultsTo(".*");
+            topicsIncludeOpt = parser.accepts("topics-include", "List of topics to verify replica consistency.")
+                .withRequiredArg()
+                .describedAs("Java regex (String)")
+                .ofType(String.class)
+                .defaultsTo(".*");
+            initialOffsetTimeOpt = parser.accepts("time", "Timestamp for getting the initial offsets.")
+                .withRequiredArg()
+                .describedAs("timestamp/-1(latest)/-2(earliest)")
+                .ofType(Long.class)
+                .defaultsTo(-1L);
+            reportIntervalOpt = parser.accepts("report-interval-ms", "The reporting interval.")
+                .withRequiredArg()
+                .describedAs("ms")
+                .ofType(Long.class)
+                .defaultsTo(30_000L);
+            options = parser.parse(args);
+            if (args.length == 0 || options.has(helpOpt)) {
+                CommandLineUtils.printUsageAndExit(parser, "Validate that all replicas for a set of topics have the same data.");
+            }
+            if (options.has(versionOpt)) {
+                CommandLineUtils.printVersionAndExit();
+            }
+            CommandLineUtils.checkInvalidArgs(parser, options, bootstrapServerOpt, brokerListOpt);
+            CommandLineUtils.checkInvalidArgs(parser, options, topicsIncludeOpt, topicWhiteListOpt);
+            if (!options.has(bootstrapServerOpt) && !options.has(brokerListOpt)) {
+                CommandLineUtils.printUsageAndExit(parser, format("The %s option is required", bootstrapServerOpt));
+            }
+            if (options.has(brokerListOpt)) {
+                System.out.printf("WARNING: The %s option is deprecated and will be removed. " +
+                    "Use the %s option with the same syntax.%n", brokerListOpt, bootstrapServerOpt);
+            }
+            if (options.has(topicWhiteListOpt)) {
```

Comment:
Did we have these warnings previously?

Reply:
No, it was just a natural consequence of adding `--bootstrap-server`. This is also mentioned in the [KIP-999](https://cwiki.apache.org/confluence/display/KAFKA/KIP-499+-+Unify+connection+name+flag+for+command+line+tool). Although it is not clear what mark as deprecated means in practice. I just feel that printing a message is nicer to users. Again, I can remove this if you prefer.

Reply:
Removing extra changes.